modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts

```diff
@@ -67,8 +67,8 @@ interface AttrHooks {
 const setupAttrHooks = (styles: Styles, settings: Partial<DOMUtilsSettings>, getContext): AttrHooks => {
   const keepValues: boolean = settings.keep_values;
   const keepUrlHook = {
-    set: ($elm, value: string, name: string) => {
-      if (settings.url_converter) {
+    set: ($elm, value: string | null, name: string) => {
+      if (settings.url_converter && value !== null) {
```

Comment:
```suggestion
      if (settings.url_converter && Type.isNonNullable(value)) {
```

Reply:
I wanted to keep this the same as below for now. If we're changing this then we probably need to change a whole lot more.

Reply:
As discussed we're going to leave this as is for now at least.